app/containers/TextInput/FormTextInput.tsx

```diff
@@ -58,112 +57,91 @@ export interface IRCTextInputProps extends TextInputProps {
 	containerStyle?: StyleProp<ViewStyle>;
 	inputStyle?: StyleProp<TextStyle>;
 	inputRef?: React.Ref<RNTextInput>;
-	testID?: string;
 	iconLeft?: TIconsName;
 	iconRight?: TIconsName;
 	left?: JSX.Element;
-	onIconRightPress?(): void;
-	theme: TSupportedThemes;
 }
 
-interface IRCTextInputState {
-	showPassword: boolean;
-}
-
-export default class FormTextInput extends React.PureComponent<IRCTextInputProps, IRCTextInputState> {
-	static defaultProps = {
-		error: {},
-		theme: 'light'
-	};
-
-	state = {
-		showPassword: false
-	};
+const FormTextInput = React.memo(
```

Comment:
This memo isn't doing anything.
TextInput doesn't handle its own state, so it receives props all the time, which is going to trigger re-renders anyway.
Even if it wasn't the case, it receives non-primitive values which also would trigger re-renders.